packages/components/src/mobile/audio-player/index.native.js

```diff
@@ -70,14 +71,30 @@ function Player( {
 		}
 	};
 
-	const containerStyle = getStylesFromColorScheme(
+	const useEditorColorScheme = ( baseStyle, darkStyle ) => {
```

Comment:
`useEditorColorScheme` provides functionality that I presume might be needed on other blocks, as choosing styles based on the background seems a common issue across blocks. I wonder if we should move it as a utility related to global styles. WDYT?

Reply:
That's a good point! I've gone ahead to refactor the function in https://github.com/WordPress/gutenberg/pull/55627/commits/190bbd07eb174150b09f0beb1da04404c66030ba, with that reusability in mind.

I also wanted to highlight that this _might_ be a short-term patch, as I plan to re-open the discussion of how we handle block-based/dark-mode colours in the long-term with the team.